File: proj/FkAndroid/fkandroid/src/main/java/com/alimin/fk/utils/FkLogcat.java
package com.alimin.fk.utils;

public class FkLogcat {
    public static void v(String tag, String msg) {
        nativeLog(0, tag, msg);
    }

    public static void d(String tag, String msg) {
        nativeLog(1, tag, msg);
    }

    public static void i(String tag, String msg) {
        nativeLog(2, tag, msg);
    }

    public static void w(String tag, String msg) {
        nativeLog(3, tag, msg);
    }

    public static void e(String tag, String msg) {
        nativeLog(4, tag, msg);
    }

    public native static void nativeLog(int type, String tag, String msg);
}

File: proj/FkAndroid/fkandroid/src/main/java/com/alimin/fk/utils/FkLogcat.kt
package com.alimin.fk.utils

class FkLogcat {
    companion object {
        fun v(tag: String, msg: String) {
            nativeLog(0, tag, msg)
        }

        fun d(tag: String, msg: String) {
            nativeLog(1, tag, msg)
        }

        fun i(tag: String, msg: String) {
            nativeLog(2, tag, msg)
        }

        fun w(tag: String, msg: String) {
            nativeLog(3, tag, msg)
        }

        fun e(tag: String, msg: String) {
            nativeLog(4, tag, msg)
        }

        external fun nativeLog(type: Int, tag: String, msg: String)
    }
}